OpenRA.Game/Sound/Sound.cs

```diff
@@ -56,26 +66,12 @@ static ISoundSource LoadSoundRaw(byte[] rawData, int channels, int sampleBits, i
 			return soundEngine.AddSoundSourceFromMemory(rawData, channels, sampleBits, sampleRate);
 		}
 
-		static ISoundEngine CreateEngine(string engine)
+		public static void Initialize(SoundSettings soundSettings, ServerSettings serverSettings)
```

Comment:
It's pretty bogus that our lowest level sound code cares about the server settings.